style.css

```diff
@@ -0,0 +1,392 @@
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  -webkit-backface-visibility: hidden; /* Safari */
+  backface-visibility: hidden;
+}
+
+.card-front {
+  background: url(img/gameOver.png);
+  transform: rotateY(180deg);
+}
+
+.card-front_bug {
+  background: url(img/bag.png);
+  transform: rotateY(180deg);
+} 
+
+/* Медиазапросы */
+@media (max-width: 1350px) {
+  .flip-three {
+    width: 65.5%;
+  }
+
+  .flip-six {
+    width: 50.5%;
+  }
+
+  .flip-ten {
+    width: 77.5%;
+  }
+
+  .flip-card,
+  .flip-card-inner,
+  .card-back,
+  .card-back__wrap,
+  .card-front {
+    background-size: 165px 261px;
+    height: 261px;
+    width: 165px;
+  }
+}
+
+@media (max-width: 1150px) {
+  .flip-ten {
+    width: 90.5%;
+  }
+}
+
+@media (max-width: 1024px) {
+  .flip-six {
+    width: 42.5%;
+  }
+
+  .flip-ten {
+    width: 64.5%;
+  }
+
+  .flip-card,
+  .flip-card-inner,
+  .card-back,
+  .card-back__wrap,
+  .card-front {
+    background-size: 105px 161px;
+    height: 161px;
+    width: 105px;
+  }
+}
+
+@media (max-width: 768px) {
+  body {
+    min-height: 815px;
+  }
+
+  .level {
+    margin-bottom: 8.8%;
+  }
+
+  .level__choice {
+    font: normal bold 54px/ 60px 'Roboto', sans-serif;
+    padding: 12% 0 4% 2.4%;
+  }
+
+  .level__difficulty {
+    font: normal bold 15px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game-level {
+    font: normal bold 46px/ 59px 'Roboto', sans-serif;
+  }
+
+  .active-level::before {
+    right: 104.4%;
+  }
+
+  .begin {
+    display: flex;
+  }
+
+  .begin__button {
+    font: normal bold 28px/ 30px 'Roboto', sans-serif;
+    width: 302px;
+    height: 92px;
+    margin: 14% 0 0 4.5%;
+  }
+
+  .card {
+    display: none;
+  }
+
+  .flip-three {
+    min-height: 815px;
+  }
+
+  .flip-six {
+    width: 55.5%;
+  }
+
+  .flip-ten {
+    width: 70.5%;
+  }
+
+  .flip-card,
+  .flip-card-inner,
+  .card-back,
+  .card-back__wrap,
+  .card-front {
+    background-size: 86px 131px;
+    height: 131px;
+    width: 86px;
+  }
+}
+
+@media (max-width: 375px) {
+  body {
+    min-height: 700px;
+  }
+
+  .level {
+    margin-bottom: 12.8%;
+  }
+
+  .level__choice {
+    font: normal bold 44px/ 55px 'Roboto', sans-serif;
+  }
+
+  .level__difficulty {
+    font: normal bold 12px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game-level {
+    font: normal bold 36px/ 59px 'Roboto', sans-serif;
+  }
+
+  .flip-three {
+    min-height: 670px;
+  }
+
+  .active-level::before {
+    width: 20px;
+    height: 20px;
+    top: 33%;
+  }
+
+  .begin__button {
+    font: normal bold 22px/ 30px 'Roboto', sans-serif;
+    width: 252px;
+    height: 72px;
+  }
+}
+
+@media (max-width: 320px) {
+  body {
+    min-height: 570px;
+  }
+
+  .level {
+    margin-bottom: 7.8%;
+  }
+
+  .level__choice {
+    font: normal bold 36px/ 45px 'Roboto', sans-serif;
+  }
+
+  .level__difficulty {
+    font: normal bold 10px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game-level {
+    font: normal bold 32px/ 59px 'Roboto', sans-serif;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5876445)